### PR TITLE
EP-1101 Order API docs

### PR DESCRIPTION
**Pull request by miguel-evocalize**
Includes these APIs:
- Place Purchase/Subscription Order
- Edit Purchase/Subscription Order
- Cancel Order
- Get Order By ID
- Get Order Refund Preview By ID

also include google in audience section

### CODE REVIEW

File: source/includes/_orders.md [hunk 1]
```diff
@@ -0,0 +1,773 @@
...
+> Place Purchase Order Response
+
+```json
+{
+  "data": {
+    "id": "1234567890",
+    "userId": "test_user_id",
+    "groupId": "test_group_id",
+    "productCode": "pid_1234567890",
+    "status": "pending",
```
Comment (sharatvisweswara) [inline source/includes/_orders.md:39]: ah I used UPPERCASE enums for this status. i can turn it into lowercase if needed, all the other enum fields like orderType and subscription status are using lowercase too may help to be consistent

File: source/includes/_orders.md [hunk 2]
```diff
@@ -0,0 +1,773 @@
...
+- If `contentItemIds` are specified, Evocalize will provide the necessary creative data using those content item IDs for
+  the product.
+- Certain variables may become immutable after placing the order due to specific product requirements. Make sure to
+  manage these immutable variable values correctly when placing the order.
+
+**Response Codes**:
+
+- `200 OK` - Successfully created the purchase order.
+- `400 BAD REQUEST` - Request is malformed or contains invalid data.
+- `404 NOT FOUND` - A specified ID does not exist.
```
Comment (sharatvisweswara) [inline source/includes/_orders.md:108]: don't think we return 404 for create order anywhere?

File: source/includes/_orders.md (hunk 1 above)
Comment (miguel-evocalize) [reply source/includes/_orders.md:39]: sounds good to me.

*also, per your comment on the other PR, we're keeping it lowercase.

Comment (miguel-evocalize) [reply source/includes/_orders.md:39]: I'll also change it so all order statuses match correctly with your updates. Currently, I have it set to whatever `order_item.status` is

File: source/includes/_orders.md [hunk 3]
```diff
@@ -0,0 +1,773 @@
...
+  format `pid_<PRODUCT_ID>`.
+
+### Scheduling
+
+- `startTimestamp` must precede `endTimestamp` by at least 24 hours.
+- `startTimestamp` should be a future date and cannot be set to a date that has already passed.
+
+### Payment
+
+- `paymentAmount` should be denominated in USD, formatted to two decimal places, and pass the minimum and maximum spend
```
[the remark this answers is not in the document]
Comment (miguel-evocalize) [reply source/includes/_orders.md:92]: Ah this is an old sentence. I had replaced it in the edit order payment section but forgot to update it here. 

Would this do? Its whats in the edit order section.
```
- `paymentAmount` should be denominated in USD, rounded to two decimal places, and pass the spend requirements. For more
  details regarding spend requirements for a specific product, contact your Client Success representative.
 ```

**Comment by miguel-evocalize** [comment]
> This is looking great!
> 
> Will we have error code enums as well that we add to this documentation or did I miss them in the PR?

Doh! I forgot the error codes. Adding them now

**Review by daniellawless — APPROVED** [review]
This looks great - thanks for the quick updates!

**Pull request merged**